packages/experiments/src/components/Button/docs/ButtonDos.md

```diff
@@ -1,11 +1,9 @@
-- Make sure the label conveys a clear purpose of the button to the user.
-- Button labels must describe the action the button will perform and should include a verb. Use concise, specific, self-explanatory labels, usually a single word.
-- Buttons should always include a noun if there is any room for interpretation about what the verb operates on.
+- Use Button labels that have a clear purpose. Labels should describe the action of the button in a single line and include a concise verb. If their is room for interpretation about what the verb operates on, a noun should also be used.
 - Consider the effect localization will have on the button and what will happen to components around it.
-- If the button’s label content is dynamic, consider how the button will resize and what will happen to components around it.
+- Consider how the button will resize if the button’s label content is dynamic.
 - Use only a single line of text in the label of the button.
-- Expose only one or two buttons to the user at a time, for example, "Accept" and "Cancel". If you need to expose more actions to the user, consider using checkboxes or radio buttons from which the user can select actions, with a single command button to trigger those actions.
+- Expose only one or two buttons to the user at a time. If you need to expose more actions to the user, consider using checkboxes or radio buttons.
```

Comment:
Replace buttons with checkboxes or radio buttons?

Remove...

Reply:
It made more sense the way it was before, but still seems a bit odd.

Reply:
I don't think replacing a button with a radio button or a checkbox makes sense. They do not execute actions. They allow the user to select a state.